fix/#80 headings in richtext

nickmoreton opened the pull request:
Removed the default config for the headings `h2` - `h6`. These headings are now parsed into a rich_text block

I have left `h1` in the config so that they are built as a separate block.

nickmoreton committed:
change the default config so that h2 - h6 are no included in a rich text bock

It felt like h1 should be left in place as a heading block as a  default. Budgets are sexy don't use h1 anywhere in body content so this should work well.

But I think as a default for other developers h1 is good as a heading block here.

nickmoreton committed:
change tests and fixtures to reflect new heading defaults

nimasmi reviewed (approved):
These changes look fine in themselves. Was there nowhere that we have example HTML and could include a test that there is a `<h2>` tag in a paragraph block?

nickmoreton committed:
add waring to fixture

The fixture is fragile in that blocks are built based on the order of the content as a whole. Changing the order breaks many tests!

nickmoreton committed:
add test for rich text block

Test a known block at position 3 as rich text. It should contain p ul and h2 tags

nimasmi reviewed (commented):
Thanks. The test looks great.
